src/material/chips/chip-grid.ts

```diff
@@ -130,6 +130,12 @@ export class MatChipGrid
    */
   readonly controlType: string = 'mat-chip-grid';
 
+  /**
+   * Implemented as part of MatFormFieldControl.
+   * @docs-private
+   */
+  readonly shouldHaveFocusOverlay = false;
```

Comment:
This shouldn't be necessary, because the form field sets a class depending on the projected form control. E.g. in this case the form field will have the `mat-mdc-form-field-type-mat-chip-grid` class which you can use like this:

```
.mat-mdc-form-field-type-mat-chip-grid .mat-mdc-form-field-focus-overlay {
  display: none;
}
```